ui/src/app/views/dataset-page/component.tsx

```diff
@@ -38,8 +38,9 @@ import {
   ValueSet,
   WorkspaceAccessLevel,
 } from 'generated/fetch';
-
-
+import {DataSetPreviewList} from 'generated/model/dataSetPreviewList';
```

Comment:
I'd make this line import from fetch:
`import {DataSetPreviewList} from 'generated/fetch'`